src/metabase/models/params/field_values.clj

```diff
@@ -108,19 +108,21 @@
   list of values and human_readable_values of the full FieldValues of the same field."
   [fv-type field hash-key constraints]
   (when-let [{:keys [values has_more_values]} (fetch-advanced-field-values fv-type field constraints)]
-    (let [;; If the full FieldValues of this field has a human-readable-values, fix it with the new values
+    (let [;; each field value in `values` is 1-vector, so unwrap them for storage
```

Comment:
wdym by `1-vector`? you mean `a vector`?

Reply:
`1-vector` Meaning a vector with one element. tbh I'm not sure if I made the word up myself, but I feel like I've seen it before. I changed this to 1-tuple, because I think that is more commonly understood as a list-like data structure with one element in it, and the fact that it's a vector doesn't matter here.